Source/WTF/wtf/text/ParsingUtilities.h

```diff
@@ -240,13 +290,16 @@ template<typename CharacterType, std::size_t Extent> constexpr bool skipCharacte
     return true;
 }
 
-// Adapt a char16_t-predicate to an Latin1Character-predicate.
+// Adapt a char16_t-predicate to an Latin1Character-predicate and char8-predicate.
```

Comment:
```suggestion
// Adapt a char16_t-predicate to an Latin1Character-predicate and char8-predicate.
```

Reply:
Now that I am reading the code, I am looking hard at it and I am uncapable of seeing what change I should make. The char8-predicate is in the code already.

Reply:
I guess you mean not updating the comment because I should remove the predicate alltogether.

Reply:
Oops, my comment was wrong. I meant to write this:
```suggestion
// Adapt a char16_t-predicate to an Latin1Character-predicate and char8_t-predicate.
```
But this is irrelevant since the concept of a char8_t adapter doesn’t quite work and we have to rethink this.